lib/components/badge/badge.dart

```diff
@@ -2,15 +2,15 @@ import 'package:flutter/material.dart';
 import 'package:tarka_ui/styles/theme.dart';
 
 enum TUIBadgeSize {
-  xs,
   s,
+  m,
   l;
```

Comment:
Can we rename those enums to 12, 16, and 24 respectively? This will help in establishing 1 to 1 mapping with the Figma design.